web/script/index.js

```diff
@@ -2,7 +2,10 @@ fetch('https://api.github.com/repos/R2Northstar/Northstar/releases/latest')
     .then(response => response.json())
     .then(data => {
         document.querySelectorAll('.download').forEach(d => { d.href = data.assets[0].browser_download_url });
-        console.log("Successfully fetched latest release download URL. Updating download links.")
+        console.log("Successfully fetched latest release download URL. Updating download links.");
+        var filesize = data.assets[0].size;
```

Comment:
i suggest using camelCase to make variable names a bit more readable

Reply:
Renamed both variables with camelCase